### PR TITLE
Account display updated

### PR DESCRIPTION
**Pull request by maxiaoping**
Account display updated

### CODE REVIEW

File: app/src/main/java/com/seafile/seadroid2/account/Account.java [hunk 1]
```diff
@@ -26,20 +27,24 @@
     public String token;
     public String sessionKey;
 
-    public Account(String server, String email, String token, Boolean is_shib) {
+    public Account(String name,String server, String email, String token, Boolean is_shib) {
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/account/Account.java]: name 参数应该放在最后

File: app/src/main/java/com/seafile/seadroid2/account/Account.java [hunk 2]
```diff
@@ -90,7 +97,7 @@ public void setSessionKey(String sessionKey) {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(server, email);
+        return Objects.hashCode(server, email,name);
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/account/Account.java]: 缺少空格

File: app/src/main/java/com/seafile/seadroid2/account/Account.java [hunk 3]
```diff
@@ -56,7 +61,9 @@ public String getServerDomainName() {
     public String getEmail() {
         return email;
     }
-
+    public String getName(){
+        return name;
+    }
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/account/Account.java]: 这里少空行

**Comment by Cisco30** [comment]
with the new update 2.2.15 it is impossible to change account

**Comment by freeplant** [comment]
@Cisco30 What Android system do you use?

**Comment by Cisco30** [comment]
Android 9.0

**Comment by freeplant** [comment]
It works on my Android 9.0. Can you try to uninstall the app and re-install it again?

**Comment by Cisco30** [comment]
the application works after the reinstallation thank you

**Pull request merged**